share/completions/az.fish

```diff
@@ -0,0 +1,16 @@
+function __fish_az_complete
+  set -x _ARGCOMPLETE 1
+  set -x _ARGCOMPLETE_IFS \n
+  set -x _ARGCOMPLETE_SUPPRESS_SPACE 1
+  set -x _ARGCOMPLETE_SHELL fish
+  set -x COMP_LINE (commandline -p)
+  set -x COMP_POINT (string length (commandline -cp))
+  set -x COMP_TYPE
+  if set -q _ARC_DEBUG
```

Comment:
We're not them, no need to keep the debugging guff.

Reply:
Done